modules/inmarBidAdapter.js

```diff
@@ -40,7 +41,7 @@ export const spec = {
       uspConsent: bidderRequest.uspConsent,
       currencyCode: config.getConfig('currency.adServerCurrency'),
       coppa: config.getConfig('coppa'),
-      firstPartyData: config.getLegacyFpd(bidderRequest.ortb2),
+      firstPartyData: getLegacyFpd(bidderRequest.ortb2),
```

Comment:
@zandree-owneriq we welcome your feedback on if you still need the data formatted this way or if you would prefer the newer ortb2 formatting?